src/adapters/supabase/helpers/user.ts

```diff
@@ -17,10 +17,8 @@ export class User extends Super {
     if ((error && !data) || !data.wallets?.address) {
       this.context.logger.error("No wallet address found", { userId, issueNumber });
       if (this.context.config.startRequiresWallet) {
-        await addCommentToIssue(this.context, this.context.config.emptyWalletText);
+        await addCommentToIssue(this.context, this.context.logger.error(this.context.config.emptyWalletText, { userId, issueNumber }).logMessage.diff);
```

Comment:
wouldn't it be better to use logger and throw?

Reply:
You mean instead of posting a comment?

or you mean log `const log = ..., addComment(log.logmessage.diff), throw new Error(log.raw)`?